plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -411,3 +412,9 @@ func TestVMEtnaActivatesCancun(t *testing.T) {
 		})
 	}
 }
+
+// currentRules returns the chain rules for the current block.
+func currentRules(vm *VM) extras.Rules {
```

Comment:
We don't necessarily need to do this, but we could avoid any other changes in the tests if we just did:
```suggestion
func (vm *VM) currentRules() extras.Rules {
```
which aligns with the prior implementation.